web_development_101/installations/installation_overview.md

```diff
@@ -14,7 +14,7 @@ In the next few lessons, we will walk through these installation steps together:
 
 * installing the [operating system](https://en.wikipedia.org/wiki/Operating_system) (OS) of your choice;
 * installing a code editor;
-* creating an SSH key (a personal "password" that will identify you to GitHub, Heroku, and many other sites you'll be using);
+* creating an [SSH key](https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys-2) (a personal "password" that will identify you to GitHub, Heroku, and many other sites you'll be using);
```

Comment:
Added a link to Digital Ocean's site on the explanation of SSH protocol and how to set up the keys.